README.md

```diff
@@ -1,4 +1,4 @@
-[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg?maxAge=3600)](https://raw.githubusercontent.com/mpusz/units/master/LICENSE)
+[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg?maxAge=3600)](https://raw.githubusercontent.com/mpusz/units/master/LICENSE.md)
```

Comment:
Shouldn't it be simply like that?
```suggestion
[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg?maxAge=3600)](./LICENSE.md)
```
Should work nicely for branches and forks.

Reply:
Oh! I didn't know that would work as well. Let @mpusz decide.

Reply:
I also did not know that it works. Let's do like @i-ky suggests.